pkg/api/v1beta2/dynakube/convert_to_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/status"
-	v1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	v1beta3 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
(or change it to have the version in the v1beta1 package)
```suggestion
	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```